### ISSUE TITLE
GitHub issue / pull request urls show "Clone with MGit"

### ISSUE THREAD

**Issue by kaeptmblaubaer1000** [maks/MGit#332]
I haven't tested it yet, but GitLab, BitBucket and NotABug are probably affected too.

**Comment by maks** [comment]
Yes I've noticed this too, but it will be difficult to come up with a path pattern(s) to effectively filter out issue/PR/wiki/etc URLs

**Comment by maks** [comment]
Because it's not possible to exclude specific url patterns in Intent Filters, it may just be better for MGit not to register this Intent filter at all.
